majifix311/src/main/java/com/example/majifix311/api/MajiFixAPI.java

```diff
@@ -138,9 +138,9 @@ public void accept(List<Problem> list, Problem problem) throws Exception {
 
     private String getAuthToken() {
         //TODO obfuscate token
-        String authToken = "Bearer "+ BuildConfig.APP_TOKEN;
+        String authToken = "Bearer "+ BuildConfig.MAJIFIX_API_TOKEN;
         System.out.println("Auth token: "+authToken);
```

Comment:
I think it best to use a logger that can work both in `android` and `jvm` environment. Lets review

1. [https://github.com/JakeWharton/timber](https://github.com/JakeWharton/timber)
2. [https://github.com/zserge/log](https://github.com/zserge/log)

Or write one.